grafana/grafana
# logs: better names and comments

this pull request has no functional changes, we only change variable-names, function-names and comments, to better explain what's going on in the code.

the current version of the comments calls all "extra fields from the dataframe" "derived fields" (a concept done in Loki), which is only correct when the data comes from Loki. for example, when the data comes from Elasticsearch, those "extra fields from the dataframe" are real data, coming from the database.

i tried to adjust the naming and comments to better explain what's going on.

if you have any idea how to improve it even further, please tell me. thanks!

how to test:
- if you review it and agree that there are no functional changes, not much testing is needed
- maybe just run the explore-page for a logs-datasource, and see if opening the log-details still shows the correct data
